packages/fetchai/connections/oef/connection.py

```diff
@@ -534,7 +534,7 @@ def on_dialogue_error(
         envelope = Envelope(
             to=self.address,
             sender=DEFAULT_OEF,
-            protocol_id=OefSearchMessage.protocol_id,
+            protocol_id=DefaultMessage.protocol_id,
```

Comment:
So technically, we now need to bump the version number of the oef connection. @marcofavorito should we bump it with each release which implements fixes? It is a lot of bumping potentially but also the cleanest way.

Reply:
yes I think that's a good approach.